src/modules/users/Config.cpp

```diff
@@ -279,6 +290,30 @@ guessProductName()
     }
     return dmiProduct;
 }
+#ifdef HAVE_ICU
+static QString
+transliterate( const QString& input )
+{
+    static auto ue = UErrorCode::U_ZERO_ERROR;
+    static auto transliterator = std::unique_ptr< icu::Transliterator >(
+                icu::Transliterator::createInstance( TRANSLITERATOR_ID, UTRANS_FORWARD, ue )
+                );
+
+    if( ue != UErrorCode::U_ZERO_ERROR ){
+        cWarning() << "Can't create transliterator";
+
+        //it'll be checked later for non-ASCII characters
+        return input;
+    }
+
+    auto transliterable = icu::UnicodeString( input.utf16() );
+
+    transliterator->transliterate( transliterable );
+
+    return QString::fromUtf16( transliterable.getTerminatedBuffer() );
+
+}
+#endif
```

Comment:
How about adding a
```
#else
static QString transliterate( const QString& input ) { return input; }
```
here so that the API is available (internally, since it's static) always? Then subsequent code is simpler.